src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -1,219 +1,232 @@
-                aria-label="Projects"
-                className={classes.roundButton}
-                color={key.startsWith('/projects') ? 'primary' : 'secondary'}
-                data-test="calendar-button"
-                size="large"
-              >
-                <Event />
-              </IconButton>
-            </NextLink>
-          </ListItem>
+          )}
+          {open && (
+            <ZUIFuture future={model.getOrganization(orgId)}>
+              {(data) => {
+                return (
+                  <Box alignSelf="center">
+                    <Box display="flex" justifyContent="start">
+                      {hover ? (
+                        <IconButton onClick={toggleDrawer}>
+                          <ChevronLeftIcon
+                            onClick={handleDrawerToggle}
+                            sx={{ alignSelf: 'center', marginLeft: 2 }}
+                          />
+                        </IconButton>
+                      ) : (
+                        <Avatar
+                          alt="icon"
+                          src={`/api/orgs/${orgId}/avatar`}
+                          sx={{ alignSelf: 'center', marginLeft: 2 }}
+                        />
+                      )}
+                      <Typography gutterBottom m={1} variant="h6">
+                        {data.title}
+                      </Typography>
+                    </Box>
+                  </Box>
+                );
+              }}
+            </ZUIFuture>
+          )}
         </Box>
-      </List>
+        <Divider />
+        <List>
+          {menuItemsMap.map((item) => (
+            <ListItem
+              key={item.name}
+              disableGutters
+              disablePadding
+              sx={{
+                display: 'block',
+              }}
+            >
+              <NextLink href={`/organize/${orgId}/${item.name}`} passHref>
+                <IconButton
+                  className={classes.roundButton}
+                  size="large"
+                  sx={{
+                    justifyContent: open ? 'initial' : 'center',
+                    minHeight: 48,
+                    ml: open ? 1 : 2,
+                    mr: open ? 2 : 1,
+                    px: 2.5,
+                  }}
+                >
+                  <Avatar
+                    sx={{
+                      backgroundColor: key.startsWith('/' + item.name)
+                        ? theme.palette.grey[300]
+                        : 'transparent',
+                      borderRadius: '20%',
+                      color: theme.palette.grey[500],
+                    }}
+                    variant="square"
+                  >
+                    {item.icon}
+                  </Avatar>
+                  <ListItemText
+                    primary={messages.organizeSidebar[item.name]()}
+                    sx={{ marginLeft: 2, opacity: open ? 1 : 0 }}
+                  />
+                </IconButton>
```

Comment:
According to the MUI documentation, an `IconButton` is only used when wanting to create a button with nothing but an icon in it. Here we have a "button" with an icon and a text.

In Figma, it looks to me like this is no button at all, but maybe it could still be achieved using a `Button` with a `startIcon`. I'm not sure what's best, so lets talk about it and experiment a little.

But right now, it doesn't look right according to the design. I can see at least the following issues:

* Margins are incorrect: Everything is too far in from the left
* Hover state is incorrect:
  * When open: The text turns red, but should not change, while the background doesn't change but should become grey (with a slight margin and rounded corners).
  * When closed: There should be a rounded rectangle background around the hovered icon, similar to the one shown for the active icons